src/stick_table.c

```diff
@@ -1048,7 +1066,7 @@ struct stktable_data_type stktable_data_types[STKTABLE_DATA_TYPES] = {
 	[STKTABLE_DT_BYTES_OUT_RATE]= { .name = "bytes_out_rate", .std_type = STD_T_FRQP, .arg_type = ARG_T_DELAY },
 	[STKTABLE_DT_GPC1]          = { .name = "gpc1",           .std_type = STD_T_UINT  },
 	[STKTABLE_DT_GPC1_RATE]     = { .name = "gpc1_rate",      .std_type = STD_T_FRQP, .arg_type = ARG_T_DELAY  },
-	[STKTABLE_DT_SERVER_NAME]   = { .name = "server_name",    .std_type = STD_T_DICT  },
+	[STKTABLE_DT_SERVER_KEY]    = { .name = "server_key",     .std_type = STD_T_DICT  },
```

Comment:
I'm repurposing the server_name data type as server_key which contains a string identifying the server, but the exact source of that string depends on the `server_key_type` field. This isn't the only way this could be done. We could instead create a new `STKTABLE_DT_SERVER_ADDR` data type, and either select that using a `store server_addr` in the stick-table directive, or have some other syntax to select the server_addr data type rather than the server_name data type. I actually started out doing something like that, but it ended up being more complicated, because more areas needed to know which data type to lookup and store the server identifier in.